content/docs/prerequisites/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Prerequisites"
+linkTitle: "Prerequisites"
+description: Prerequisites for CSM 
+weight: 1
+---
+
+## Supported Storage Platforms
```

Comment:
Also, there is no matrix to reveal co-relation between modules and drivers - Which modules are supported by each driver?

Reply:
Look at the notes section

Reply:
It was easier to call out what was not supported